### DISCUSSION TITLE
Inline Editable and Column Resizing Functionality

### DISCUSSION THREAD

**Discussion by PointSingularity** [icflorescu/mantine-datatable#27] [Ideas]
Hey thanks for the great library!

Couple of ideas/suggestions: 
- Option to be able to resize columns width with a drag and click would be great
- Option to edit cells inline

Any plans for any of these, or are they out of the scope for this library?

**Comment by icflorescu** [comment]
Thanks for the kind words!

> Any plans for any of these, or are they out of the scope for this library?

Not at the moment, I'm afraid.
Inline cell editing, especially, would be way too complex for the scope of this package IMO.
But if you believe otherwise, feel free to have a go at it ;-)

**Reply by PointSingularity** [reply]
That is understandable, and if I find some time in the future I might try it 😄 

Thank you again!

**Reply by icflorescu** [reply]
Have a look at this discussion too: https://github.com/icflorescu/mantine-datatable/discussions/28
